CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.8.0 (October 18, 2022). Tested on Artifactory 7.46.6 and Xray 3.59.4
+
+IMPROVEMENTS:
+
+* resource/xray_watch: add functionality to apply `path_ant_filter` for `repository` and `all-repos` `watch_resource.type`.
+ [#PR](https://github.com/jfrog/terraform-provider-xray/pull/82)
```

Comment:
Shouldn't this be in the format `PR [#82](https://github.com/jfrog/terraform-provider-xray/pull/82)`?

Reply:
yes, typo. Fixed.